packages/stencil-component/readme.md

```diff
@@ -18,17 +18,22 @@ you make to `/src` will be automatically shown in the browser.
-| `theme`           | `theme`             | Theme to use: ['light', 'dark'] defaults to 'dark'                                                     | `string`                     | `'light'`      |
-| `useAssetsFolder` | `use-assets-folder` | Toggle the use of assets folder for resolving image urls. Defaults to 'true' for backwards compatibility   | `boolean`              | `true`         |
+| Property             | Attribute                | Description                                                                                                      | Type            | Default     |
+|----------------------|--------------------------|------------------------------------------------------------------------------------------------------------------|-----------------|-------------|
+| `alignment`          | `alignment`              | The alignment as SMIL                                                                                            | `string`        | `undefined` |
+| `audio`              | `audio`                  | The audio file                                                                                                   | `string`        | `undefined` |
+| `cssUrl`             | `css-url`                | Optional custom Stylesheet to override defaults                                                                  | `string`        | `undefined` |
+| `language`           | `language`               | Language  of the interface. In 639-3 code Options are - "eng" for English - "fra" for French                     | `"eng" \        | "fra"`      | `'eng'`        |
+| `pageScrolling`      | `page-scrolling`         | Toggles the page scrolling from horizontal to vertical. Defaults to horizontal                                   | `"horizontal" \ | "vertical"` | `"horizontal"` |
+| `svgOverlay`         | `svg-overlay`            | Overlay This is an SVG overlay to place over the progress bar                                                    | `string`        | `undefined` |
+| `text`               | `text`                   | The text as TEI                                                                                                  | `string`        | `undefined` |
+| `theme`              | `theme`                  | Theme to use: ['light', 'dark'] defaults to 'dark'                                                               | `string`        | `'light'`   |
+| `useAssetsFolder`    | `use-assets-folder`      | Toggle the use of assets folder for resolving image urls. Defaults to 'true' for backwards compatibility         | `boolean`       | `true`      |
+| `scrollBehavior`     | `scroll-behavior`        | Toggle the use of animation during scrolling. Defaults to 'smooth' for backwards compatibility                   | `"auto  \       | "smooth"`   | `smooth`    |
```

Comment:
Is this table generated automatically from the source code? This line, as well as those for `language` and `pageScrolling`, don't render properly. see https://github.com/ReadAlongs/Web-Component/tree/patch.page_transition/packages/stencil-component

Reply:
fixed. The source had `"eng" \   | "fra"` so I removed the spaces between `\` and `|` and it's fine now.

Reply:
> Thanks for fixing the but I reported earlier! Unfortunately, I found another one.
> 
> If I click on a word on page two of an RA, and then on a second word also on page two, the view scrolls back to page one. This happens whether I have started playing the RA or not.
> 
> I just noticed this error in the console, copying it here in case it's relevant:
> 
> ```
> Uncaught TypeError: Cannot read properties of undefined (reading '0')
>     at Howl.play (read-along.entry.js:872:82)
>     at read-along.entry.js:10863:54
> play @ read-along.entry.js:872
> (anonymous) @ read-along.entry.js:10863
> setTimeout (async)
> scrollEventHandler @ read-along.entry.js:10863
> onScroll @ read-along.entry.js:11637
> 
> By the way, for manual testing, I have to use `localhost:3333`, `0.0.0.0:3333` does not work, but it looks like cypress is OK with the 0.0.0.0 because that suite passed.
> ```

the output is useful.. will take a look at it